apps/libsignpost/signpost_api.c

```diff
@@ -339,26 +335,25 @@ int signpost_initialization_module_init(uint8_t i2c_address, api_handler_t** api
         yield_for(&done);
     }
 
-    gpio_disable_interrupt(MOD_IN);
-    gpio_toggle(MOD_OUT);
-    led_toggle(RED_LED);
+    port_signpost_gpio_disable_interrupt(MOD_IN);
+    port_signpost_gpio_set(MOD_OUT);
+    //led_toggle(RED_LED);
```

Comment:
hrm.. sad-ish to see this go, maybe include a `debug_led` in the port layer that is an optional no-op